Tower_Of_Hanoi/Tower_of_Hanoi.dart

```diff
@@ -0,0 +1,46 @@
+/* Tower of Hanoi is a mathematical problem where we have three rods A, B and C and n number of disks where n is taken from user in this code. The goal of the problem is to transfer the entire stack of disks to another rod, obeying the conditions as follows:

```

Comment:
Please split this line into multiple, it is going out of window. 80 characters per line is recommendation.